### ISSUE TITLE
Jazzy could not recognize the `List-Bullet` tag.

### ISSUE THREAD

**Issue by kiary** [realm/jazzy#476]
Just got this error while generating the doc. The problem is in a comment like this

```
/**
Load accounts that are synchronized in the iPhone

Load the different sources (Facebook, Local, Gmail, Outlook,...) found in the 
Address Book of the iPhone and gives each one it appropriate name.

 - parameter addressBook: The ABAddressBook object relative to the local Address Book

 - returns: A tuple of objects:
    * An array of strings with the names of the accounts
    * An array of ABRecord with the actual sources objects relative to the names
    * A dictionary with the correspondences between accounts names and their ID 
            for future retrieval

 */
```

The list inside the returns tag is not visible in the doc.

**Comment by jpsim** [comment]
Sorry for the delay in getting back to you @kiary, I think the issue here is that jazzy expects the contents of `returns` to be a single markdown statement, not multiple ones, like an unordered list. I suppose this works well in most cases, but certainly not this one. In any case, I'd consider this a bug and something we should fix, but I'd encourage you to work around this for now.

**Comment by chillok** [comment]
@jpsim Also getting this error - though it's not in a return. Will I file a fresh bug? This is essentially the code.

```
/**
     Pushes a specific Screen from the specified UIViewController.
     
     - parameter style:              The design style of the Screen.
                                        
        ### Available Styles:
        - Screen.A
        - Screen.B
     
     - parameter fromViewController: the `UIViewController` to push from.
     */
```

**Comment by mbmadaj** [comment]
I can also confirm bullet list is not recognized by jazzy:

`
public class Test {
    /**
     Does something.

     - note:
     Some note

     - parameter nested: Any?

         Available option:
         * something

    - parameter other: String

     - returns: AnyObject?
     */
    public func xyz(nested:Any?, other: String) -> AnyObject? {
        return nil
    }
}
`

[Test.swift.zip](https://github.com/realm/jazzy/files/227345/Test.swift.zip)

![jazzy parsing result](https://cloud.githubusercontent.com/assets/1888852/14665661/ee2929c4-06d4-11e6-9e40-fd8946d1e238.png)

**Comment by bolismauro** [comment]
This works for me
```
/**
  List:
  
  - B

  - A
```

**Comment by istx25** [comment]
Any progress on this?

**Comment by jpsim** [comment]
Bug still exists.

**Comment by andyjohns** [comment]
I get this error too. (using latest jazzy v0.7.5) is there any way to tell what file/class/method is causing the error? I could at least adjust the doc comment if I could find where it's coming from.

**Comment by SDGGiesbrecht** [comment]
I think this is the same as #629. It has been investigated in more detail there.

@andyjohns, for callouts (such as `returns`, `parameter`, or `note`), only single‐line entries work right now.

**Comment by jpsim** [comment]
Fixed in #795. Thanks @johnfairh 👏
